frontend/core/blocks/image-augmentations/computations.py

```diff
@@ -0,0 +1,354 @@
+
+    def zoom_center(input_image_numpy, parameter=0.2):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[parameter, parameter], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=None, shift_limit_y=None, always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def zoom_right(input_image_numpy, parameter=1):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[parameter, parameter], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=[-parameter/2.0, -parameter/2.0], shift_limit_y=None, always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def zoom_left(input_image_numpy, parameter=1):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[parameter, parameter], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=[parameter/2.0, parameter/2.0], shift_limit_y=None, always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def zoom_top(input_image_numpy, parameter=1):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[parameter, parameter], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=None, shift_limit_y=[parameter/2.0, parameter/2.0], always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def zoom_bottom(input_image_numpy, parameter=1):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[parameter, parameter], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=None, shift_limit_y=[-parameter/2.0, -parameter/2.0], always_apply=True)])
```

Comment:
parameters are hardcoded is that okay or will there be some need for flexibility for the parameters